ui/package.json

```diff
@@ -100,6 +111,7 @@
     "eslint-plugin-prettier": "^3.0.0",
     "flat": "^4.1.0",
     "ivy-codemirror": "2.1.0",
+    "jsdoc-to-markdown": "^4.0.1",
```

Comment:
If we're not going to use these in tests for now, can we move all of the new deps to `optionalDependencies` so that we don't install all of it when doing tests or release builds?